ci-operator/step-registry/ipi/conf/aws/custom-vpc/ipi-conf-aws-custom-vpc-commands.sh

```diff
@@ -6,11 +6,40 @@ set -o pipefail
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
-CONFIG="${SHARED_DIR}/install-config.yaml"
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+RELEASE_IMAGE_INSTALL="${RELEASE_IMAGE_INITIAL:-}"
+if [[ -z "${RELEASE_IMAGE_INSTALL}" ]]; then
+  # If there is no initial release, we will be installing latest.
+  RELEASE_IMAGE_INSTALL="${RELEASE_IMAGE_LATEST:-}"
+fi
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_INSTALL} -ojsonpath='{.metadata.version}' | cut -d. -f 1,2)
+ocp_major_version=$(echo "${ocp_version}" | awk --field-separator=. '{print $1}')
+ocp_minor_version=$(echo "${ocp_version}" | awk --field-separator=. '{print $2}')
+rm /tmp/pull-secret
+
+set -x
 
+CONFIG="${SHARED_DIR}/install-config.yaml"
+REGION="${LEASED_RESOURCE}"
 if [ -n "${PUBLISH}" ] && [ X"${PUBLISH}" == X"Internal" ]; then
-  subnet_ids_file="${SHARED_DIR}/private_subnet_ids"	
+  subnet_ids_file="${SHARED_DIR}/private_subnet_ids"
   echo "This is a private cluster so use only private subnets from the VPC"
+
+  # Starting from 4.19, installer supports 
+  #  AWS - Allocate Load Balancers (API & Ingress) to Specific Subnets https://issues.redhat.com/browse/OCPSTRAT-569
+  #  A new restriction is introduced:
+  #    Do Not Use Untagged Subnets User Stories
+  #    https://github.com/openshift/enhancements/pull/1634/files#diff-ffcfdf0d21ba360a17e0ac9846e83eec8ecf0ba8b15d429d42e7de2dbd0bfaf7R151
+  #  For private cluster, only private subnets were used, 
+  #  the other subnets in the same VPC (public subnets) require tag `kubernetes.io/cluster/unmanaged`
```

Comment:
```suggestion
  #  Because of CCM subnet discovery limitation, the other subnets in the same VPC (public subnets) require tag `kubernetes.io/cluster/unmanaged`
```